web/styles/settings.css

```diff
@@ -5,6 +5,10 @@ $settings_sidebar_width: 255px;
 
 label {
     margin: 0;
+
+    &:not([for]) {
+        cursor: default;
+    }
```

Comment:
I don't love `not` CSS rules; they can cause performance issues and otherwise are hard to reason about. Is there a better way to do this, like only setting `cursor: pointer` on `label[for]`?

Reply:
Yeah, I dropped the commit since the original problem is no longer there. The issue with only `cursor: pointer` on `label[for]` is that bootstrap is applying `cursor: pointer` on every labels. So we'll have to surgically remove that before we are able to do this.